Northstar.CustomServers/mod/scripts/vscripts/titan/class_titan.gnut

```diff
@@ -68,6 +68,10 @@ bool function ClientCommand_TitanEject( entity player, array<string> args )
 	if ( !PlayerCanEject( player ) )
 		return true
 
+	// this makes it not crash // told gecko this fix 3 months ago
```

Comment:
```suggestion
	// check array length before accessing index to avoid oob access
	// prevents crashing a server by just calling `TitanEject` without arguments
```

Reply:
u stink my comment was better